sample-compose/app/build.gradle.kts

```diff
@@ -116,60 +119,49 @@ kapt {
 }
 
 dependencies {
-    implementation(project(Module.DATA))
-    implementation(project(Module.DOMAIN))
+    implementation(projects.data)
+    implementation(projects.domain)
 
     implementation(fileTree(mapOf("dir" to "libs", "include" to listOf("*.jar"))))
 
-    implementation("androidx.core:core-ktx:${Versions.ANDROIDX_CORE_KTX_VERSION}")
-    implementation("androidx.lifecycle:lifecycle-runtime-ktx:${Versions.ANDROIDX_LIFECYCLE_VERSION}")
-    implementation("androidx.lifecycle:lifecycle-runtime-compose:${Versions.ANDROIDX_LIFECYCLE_VERSION}")
+    implementation(libs.bundles.androidx)
 
-    implementation(platform("androidx.compose:compose-bom:${Versions.COMPOSE_BOM_VERSION}"))
-    implementation("androidx.compose.ui:ui")
-    implementation("androidx.compose.ui:ui-tooling")
-    implementation("androidx.compose.foundation:foundation")
-    implementation("androidx.compose.material:material")
+    implementation(platform(libs.compose.bom))
+    implementation(libs.bundles.compose)
 
-    implementation("androidx.navigation:navigation-compose:${Versions.COMPOSE_NAVIGATION_VERSION}")
-    implementation("com.google.accompanist:accompanist-permissions:${Versions.ACCOMPANIST_VERSION}")
-    implementation("com.google.accompanist:accompanist-systemuicontroller:${Versions.ACCOMPANIST_VERSION}")
+    implementation(libs.accompanist.permissions)
+    implementation(libs.accompanist.systemUiController)
```

Comment:
I think we should create new issue to deprecate this lib. Now this officially support in `androidx.activity` 💁‍♂️ https://google.github.io/accompanist/systemuicontroller/

![image](https://github.com/user-attachments/assets/2fbb46fb-e69c-4977-b92f-9e7f7261dbf3)